client/a8c-for-agencies/sections/marketplace/shopping-cart/shopping-cart-menu/item.tsx

```diff
@@ -18,6 +19,8 @@ export default function ShoppingCartMenuItem( { item, onRemoveItem }: ItemProps
 	const translate = useTranslate();
 	const userProducts = useSelector( getProductsList );
 
+	const siteId = getQueryArg( window.location.href, 'site_id' )?.toString();
```

Comment:
Hey @yashwin @andrii-lysenko 👋 
We're seeing errors regarding this code:
```'siteId' is assigned a value but never used```

We are removing it https://github.com/Automattic/wp-calypso/pull/89617/files#diff-54fcfe0880d7f84f40422281bc99e1648454bf7ea2b7bac99922b7f4acebf6bdL22, otherwise we're not good to proceed with merging our PR. Please tell us if we're missing something

Reply:
Sorry! Not sure how this got missed. 

Please go ahead 👍